pkgs/scorec.yaml

```diff
@@ -3,9 +3,9 @@ dependencies:
   build: [mpi, parmetis, zoltan]
 
 sources:
-  - url: https://github.com/SCOREC/core.git
-    key: git:e219777edd875d35096dfedd7197300b8e2f3837
-    
+  - url: https://github.com/tjcorona/core.git
```

Comment:
Have your changes to scorec core not been merged in? Maybe we should put and if statement here to do this conditionally only for darwin or do it as a patch instead. Maybe @zhanga can look into the changes on scorec core.